.github/workflows/ci-workflow.yml

```
@@ -6,20 +6,38 @@ on:
       - README.md
     branches:
       - main
-      - exercise-8  
   pull_request:
     paths-ignore:
       - README.md
     branches:
       - main
       - 'exercise**'           
 jobs:
-  docker:
+  docker-run-test:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Build container
         run: docker build --target test --tag todo-app-test .
       - name: Run container
-        run: docker run todo-app-test
+        run: docker run todo-app-test           
+  docker-run-main:
```

While I understand the naming, it won't necessarily mean much to the next person to have to work with this! Would a name more like below be clearer?
```suggestion
  docker-run-deployment:
```

---

Updated